jdbc/src/main/java/com/interface21/jdbc/core/JdbcTemplate.java

```diff
@@ -14,4 +23,89 @@ public class JdbcTemplate {
     public JdbcTemplate(final DataSource dataSource) {
         this.dataSource = dataSource;
     }
+
+    public int update(String sql, Object... args) {
+        return execute(sql, pstmt -> {
+            setParameters(pstmt, args);
+            return pstmt.executeUpdate();
+        });
+    }
+
+    public <T> T queryForObject(String sql, RowMapper<T> rowMapper, Object... args) {
+        List<T> results = query(sql, rowMapper, args);
+        
+        if (results.isEmpty()) {
+            throw new EmptyResultDataAccessException("Query returned no results");
+        }
+        
+        if (results.size() > 1) {
+            throw new IncorrectResultSizeDataAccessException(1, results.size());
+        }
+        
+        return results.get(0);
```

Comment:
```suggestion
        return results.getFirst();
```

컬렉션 프레임워크에서 제공하는 메서드를 써서 명확하게 보여줘도 좋을것 같아요

Reply:
인정